### DISCUSSION TITLE
Add a counter for revisions in the admin menu

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Revisions#914] [Feature Requests]
> Is it possible to customize the plugin so that a notification of the number of revisions awaiting validation appears in the menu ?

<img width="341" alt="Screen Shot 2020-11-23 at 10 09 01 AM" src="https://user-images.githubusercontent.com/3868028/99978430-f03dc100-2d73-11eb-9a37-af5d88b4827d.png">

**Comment by stevejburge** [comment]
Related:

> 
My colleague is wanting to know if you there is an indicator about an existing Revision when you're editing a Post. I feel like I saw this in the past, but don't see anything now. Of course she can go to the Revision Queue to check, but she was hoping there would be some place on the post edit screen itself that would show that there is an existing revision

**Comment by KratosGemini** [comment]
I'm reviewing the feature requests and thought I'd mention that the related request in the first comment seems to be an implemented feature now:

![image](https://github.com/publishpress/PublishPress-Revisions/assets/1143805/6ad52662-8e9f-4150-bf30-77fc3a66da74)

**Comment by stevejburge** [comment]
Thanks @KratosGemini. Yes, that is available now
